woocommerce/woocommerce
# CYS - make the progress bar move forward in small segments

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #41231 

- Duplicates the loader steps with incremented progress value to move the progress bar more often
- Applied transition to the progress bar

https://github.com/woocommerce/woocommerce/assets/4723145/59879c40-e988-4159-8cfc-2c35469014b4






### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch and WooCommerce Beta Tester
2. Install and activate [woocommerce-blocks-trunk-nightly.zip](https://github.com/woocommerce/woocommerce-blocks/releases/download/nightly/woocommerce-blocks-trunk-nightly.zip)
3. Enable `customize-store` feature flag on Tools -> WCA Test Helper -> Features
4. Go to `WooCommerce -> Home` and start Cutomize your store task
5. Proceed to the Tone of Voice page and click Continue
6. You should see the updated API Loader

You can also run the storybook locally.

1. Checkout this branch locally
2. Run `pnpm --filter=@woocommerce/storybook run storybook`
3. Open the storybook once it's ready and choose `Application -> Customize Store -> Design with AI -> API Call Loader -> Api Call Loader Page with Smooth Transition`




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update the CYS task API loader smoother

#### Comment 

</details>
